utils/jwt.go

```diff
@@ -9,10 +9,14 @@ import (
 
 type JWTClaims struct {
 	Audience []string `json:"aud"`
+	Features []string `json:"1password.com/fts"`
```

Comment:
Can we follow the approach of go-jose @ _vendor/gopkg.in/square/go-jose.v2/jwt/claims.go:114_ and create a `type Features []string` with a `Contain()` method?  
This will allow us to call `jwt.Features.Contains(utils.SignInAttemptsFeatureScope)` instead of making a new `Have...` for each new feature we add.